docs/develop/python/message-passing.mdx

```diff
@@ -362,54 +501,24 @@ Use static definitions as the primary way of structuring your Workflows.
 Reserve Dynamic Handlers for cases where the handler names are not known at compile time and need to be looked up dynamically at runtime.
 They are meant to handle edge cases and act as a catch-all, not as the main way of invoking logic.
 
-:::
-
-### Set a Dynamic Signal {#set-a-dynamic-signal}
-
-**How to set a Dynamic Signal**
+### Set a Dynamic Signal, Query, or Update handler {#set-a-dynamic-signal}
```

Comment:
This section only covers signal

Reply:
It's not great, but I think it's OK to merge. It shows how to use it for signal, by way of an example. The whole "dynamic XXX" topic needs more work, and wasn't the focus of this PR.